build_node/build_node_builder.py

```diff
@@ -165,6 +167,10 @@ def run(self):
                     build_artifacts = []
                     success = False
                 finally:
+                    with open(task_log_file, 'rb') as src:
```

Comment:
more short:
``` with open(one_path) as fd_1, open(two_path) as fd_2:```

Reply:
it doesn't work in python 3.6.8

Reply:
Are you sure?
```
[soksanichenko@localhost ~]$ python3.6
Python 3.6.8 (default, Nov  9 2022, 09:57:34)
[GCC 8.5.0 20210514 (Red Hat 8.5.0-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> with open('/tmp/1', 'w') as fd1, open('/tmp/2', 'w') as fd2:
...   fd1.write('1')
...   fd2.write('2')
...
1
1
>>>
[soksanichenko@localhost ~]$ cat /tmp/1
1[soksanichenko@localhost ~]$ cat /tmp/2
2[soksanichenko@localhost ~]$
```

Reply:
yup, you're right, I guess I did something wrong, thanks